README.md

```diff
@@ -1 +1,38 @@
-# Capston_Project
\ No newline at end of file
+# Capston_Project
```

Comment:
It's required to give proper credit to the original idea owner, so please make sure to include the `Credits` section in your README file.

![image](https://user-images.githubusercontent.com/34779928/196795255-60e74afe-36c7-4e23-be71-03cc54598fd9.png)